plugin/src/main.rs

```diff
@@ -54,6 +56,14 @@ impl CliArgs {
                 operations.cli_args.namespace = ns;
                 operations.cli_args.kubeconfig = path
             }
+            cli_utils::Operations::Upgrade(ref mut upgrade_args) => {
+                upgrade_args.namespace = ns;
+                upgrade_args.kubeconfig = path
+            }
+            cli_utils::Operations::Get(ref mut get_able) => {
```

Comment:
Do we need to add `Get` command as a top level verb

Cant it be after `upgrade`

For eg: `kubectl openebs upgrade get status` / ` kubectl openebs upgrade status`

Reply:
I've changed the `kubectl openebs get upgrade-status` to `kubectl openebs upgrade status`. No change to `kubectl openebs upgrade`.